presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -743,12 +743,6 @@ std::shared_ptr<connector::ConnectorTableHandle> toConnectorTableHandle(
   if (auto hiveLayout =
           std::dynamic_pointer_cast<const protocol::HiveTableLayoutHandle>(
               tableHandle.connectorTableLayout)) {
-    VELOX_USER_CHECK(
```

Comment:
Would you add an e2e test to query TEXTFILE or JSON table and verify that error message is clear?

Reply:
How do I create a table backed by JSON or TEXTFILE in e2e test?  It seems `CREATE TABLE t WITH (format = 'JSON')` would throw error because we need to implement the corresponding writers first.

Reply:
Use 'expectedQueryRunner' (Java-based). This would be similar to how we create partitioned and bucketed tables.

Reply:
See com.facebook.presto.nativeworker.NativeQueryRunnerUtils#createBucketedCustomer